src/styles.css

```
@@ -1,407 +1,407 @@
+    --sidebar-accent-foreground: oklch(0.9850 0 0);
+    --sidebar-border: oklch(0.2750 0 0);
+    --sidebar-ring: oklch(0.4390 0 0);
+
+    --radius: 0.625rem;
+}
+
+/* ============================================
+   Tailwind v4 Theme Mapping
+   ============================================ */
+
+@theme inline {
```

Stylelint (17.11.0)

[error] Unexpected unknown at-rule "`@theme`" (scss/at-rule-no-unknown)

(scss/at-rule-no-unknown)